### DISCUSSION TITLE
欢迎来到金庸群侠传3D重制版讨论区

### DISCUSSION THREAD

**Discussion by chenggongchina** [jynew/jynew#141] [Announcements]
## 欢迎

这里是讨论区，你可以发帖讨论，希望大家和谐互助友爱发言，你可以

* 关于游戏开发提问题
* 分享你的创意
* 讨论技术
* 聊天吹水

如果希望更多的被人关注和回复，也可以介绍一下你自己

**Comment by KyleVamp** [comment]
没人吗？开始讨论吧？这么好的游戏引擎，可以制作其他武侠游戏了

**Reply by chenggongchina** [reply]
需要先完善基础功能，还有很多苟且的地方。

**Reply by raysnu** [reply]
!   檔案名稱、目錄名稱或磁碟區標籤語法錯誤。
!   D:\Users\Desktop\jynew-main.zip: 無法建立 jynew-main\docs\UI鏁寸悊\涓昏彍鍗?png
!   檔案名稱、目錄名稱或磁碟區標籤語法錯誤。
!   D:\Users\Desktop\jynew-main.zip: 無法建立 jynew-main\jyx2\Assets\3D\Animation\Jyx2remastered\鍠囧槢鎸佹硶鏉?anim
!   檔案名稱、目錄名稱或磁碟區標籤語法錯誤。


**請問解壓縮出現這個如何解決?**

**Reply by easterNday** [reply]
更換壓縮包編碼文件。應該是中文導致的編碼錯誤，更換壓縮包編碼即可。

**Comment by apthskyi** [comment]
大佬, 请问, 这个项目可以在MacOS里的Unity3D跑起来吗? 我下载下来之后发现提示错误如下:

An error occurred while resolving packages:
  Project has invalid dependencies:
    com.unity.addressables.cn: connect ETIMEDOUT 35.238.21.138:443
    com.unity.cinemachine: connect ETIMEDOUT 35.238.21.138:443
    com.unity.collab-proxy: connect ETIMEDOUT 35.238.21.138:443
    com.unity.ide.visualstudio: connect ETIMEDOUT 35.238.21.138:443
    com.unity.postprocessing: connect ETIMEDOUT 35.238.21.138:443
    com.unity.purchasing: connect ETIMEDOUT 35.238.21.138:443

A re-import of the project may be required to fix the issue or a manual modification of /Users/a555025/Documents/Unity3D/jynew/jyx2/Packages/manifest.json file.

**Reply by apthskyi** [reply]
自问自答一下吧, 通过package manager替换addressables.cn 为 addressables, 并升级其他package, 终于跑起来了.
具体原因可能是因为我用的是英文版unity, 并且通过公司网络, 无法访问到.cn结尾的服务器

**Reply by chenggongchina** [reply]
好的！

**Reply by bz1983skywalker** [reply]
20210614的asset怎麼可以下載只剩兩個，原本不是四個嗎？

**Comment by mguannan** [comment]
请问如何全屏游戏？

**Reply by chenggongchina** [reply]
PC下alt+enter，现在还没实现切换分辨率设置

**Comment by DH1214** [comment]
代码库在哪 入群 找不到代码库名称=。=

**Reply by chenggongchina** [reply]
这个问题。。要让我如何回答呢

**Comment by KyleVamp** [comment]
全部使用英文路径。










在 2021-08-30 18:48:01，"raysnu" ***@***.***> 写道：

! 檔案名稱、目錄名稱或磁碟區標籤語法錯誤。
! D:\Users\Desktop\jynew-main.zip: 無法建立 jynew-main\docs\UI鏁寸悊\涓昏彍鍗?png
! 檔案名稱、目錄名稱或磁碟區標籤語法錯誤。
! D:\Users\Desktop\jynew-main.zip: 無法建立 jynew-main\jyx2\Assets\3D\Animation\Jyx2remastered\鍠囧槢鎸佹硶鏉?anim
! 檔案名稱、目錄名稱或磁碟區標籤語法錯誤。

請問解壓縮出現這個如何解決?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by char1986** [comment]
请问这里可以反馈BUG吗？如另有反馈途径麻烦告知，谢谢。
试玩了0920版，发现以下BUG。
1.目前的攻防数据有问题。最明显的是救杨过大蛇战，大蛇打我方伤害只有个位数。其他一些战斗的伤害数据也有这样的问题。这个极大的影响游戏体验啊。
2.韦小宝的物品数量买完了再对话会重置，可以买无数件软猬甲。
3.拿真武剑原版应该是大减道德的，现在好像看不出来。
4.坐船也可以直接进场景（如百花谷）导致出场景后会卡死
5.杨过小龙女回古墓后，再收入队伍，会出现两把玄铁剑。
6.部分场景空气墙缺失可以乱跑。印象深刻的是有间客栈
7.安卓版在大地图帧数只有15~20，进入场景后有60帧左右。是否尚待优化？设备为三星note9

**Reply by Colemanpig** [reply]
8. 黑木崖 战五岳派 仍有问题

**Reply by char1986** [reply]
https://www.bilibili.com/video/BV1zv411P7vr  评论区有最新版地址

**Reply by tyq132** [reply]
9.光明顶剧情后峨嵋派，昆仑派、崆峒派无法对话，虽然不影响剧情但是打灭绝拿倚天还是蛮有趣的，安卓毕现，不知是否fixed

**Comment by janehuang95** [comment]
发现几个BUG（针对打包出来的安卓版）
版本号：20211115
1. 刚刚进入场景后没有离开场景的按钮，需要离开触发判定区后再进入才可以再次触发
2. 与阎基的战斗，阎基对主角造成的中毒伤害一直为“中毒+xx，毒发+0”，但实际没有产生任何伤害
3. 蜘蛛洞的战斗，战败后连续两次在【GAMEOVER读档界面】弹出对话框（覆盖在【GAMEOVER读档界面】上方），点击对话框后，又重新进入蜘蛛洞的战斗（这时全员空血，明显是已经进行过战败结算了），第三次之后才恢复正常

**Comment by Yujian510** [comment]
版本号20211204 PC端steam版，试玩了一个半小时
程序性问题
1. 崆峒派没有后墙也没有空气墙
![屏幕截图(2)](https://user-images.githubusercontent.com/95614100/144817504-d646faec-ad46-47b6-8532-aad3f8f5ff62.png)


2. 昆仑山洞出来之后，34，347位置有一条路，实际是地图边缘了，但前面还是可以探索，不知是否是准备未来在这里加点东西

![屏幕截图(3)](https://user-images.githubusercontent.com/95614100/144817696-fbec0562-ff4a-4134-809c-f53b3c237985.png)

3. 韦小宝卖的那个加生命的药，买完之后背包里没有，怀疑他又给我偷回去了……

游戏性问题：
1. 鼠标自动寻路导致的问题
由于支持鼠标操作，也支持自动寻路出现某些奇怪的问题，最典型的就是黑龙潭，只需要鼠标点两下，徐小侠自动就走过去了，程英哭死

2. 背包用滚轮滑动速度过慢
还好我之前就养成了不用拿的东西不拿的习惯，不过找蟠桃天山雪莲也是很滑了半天的，韦小宝那个药更是来回滑了两遍都没找到……

3. 使用等待会回到原位，这个没有截到图，虽说不影响游戏但还是有点奇怪
4. 百花谷给老顽童玉峰浆后，还没出门，地图瞬间增加了蜂箱，有点鬼畜

游戏性建议：
1. 大地图的入口提示。某些地方其实不适合有提示，比如唐诗山洞，再比如绝情谷底，因为有入口提示，所以小龙女辛辛苦苦雕的“一起拔”（？！）就没有作用了……
2. 野球拳特效似乎还是太绚丽了点，至少1-9级，更朴实无华一点为妙
3. 王语嫣功能似乎弱化很多，游戏第一人遭遇史诗级削弱？
4. 更惨的是龙女，不光要雕刻羞耻的“一起拔”，这个头像，第一眼吓我一跳
![屏幕截图(1)](https://user-images.githubusercontent.com/95614100/144820285-ecf77c2c-0eaf-4e4a-bb12-fb96bddfd327.png)
5. 因为资质在创建人物的时候没法选择，所以还是建议恢复存档2存档3里的徐小侠和小虾米，哪怕是作为彩蛋，看上去是空存档，实际可以读呢。
另，真男人就是要切JJ，进入游戏第一件事就是切JJ。增加了自宫特效，很棒，就是这音效……也太烧了……

**Reply by qige2016** [reply]
感谢对该项目的关注。以下对一些问题做下回复
程序性问题：
3.这个可能是背包太乱没找到
游戏性问题：
3.王语嫣功能应该是和原版差不多的

**Comment by raysnu** [comment]
為什麼練野球拳都不會增加經驗值跟升等級
左右互搏之術主角不能學嗎???...都說不適合

**Reply by MephistoSatanDevil** [reply]
野球拳平均打60+拳才会升一级，你打了那么多了么？
左右互搏要求修炼角色资质低于45才行，你资质符合条件么？

**Comment by BoardTx** [comment]
请问steam版本如何下载 搜索不到呀

**Reply by MephistoSatanDevil** [reply]
没有了

**Comment by duduxianxian** [comment]
你好，有个问题，只要我打完铁掌山的小怪再存档，就会卡死，不论我在这中间干什么其他的事都没用，试了7、8次了也没存上档。设备是华为nova7。不知道怎么办呀

**Comment by supermansaga** [comment]
如果升级武功没那么累就更好了。用cheat engine 7.4也未能实现最快。

**Reply by MephistoSatanDevil** [reply]
都已经用CE了，那为什么不用控制台呢？有什么区别？

**Reply by supermansaga** [reply]
谢谢。之前不知道控制台大法。非常享受。就快通关。感激！

**Comment by vejita73** [comment]
> 
我在倚天屠龍記的劇情線中卡住了
目前的劇情中
已經迎回金毛獅王
要打贏光明聖火陣才能取得倚天屠龍記
但明教的紫衫龍王還未回歸
而關於紫衫龍王的部份
已經打敗過金花婆婆
讓王難姑回蝴蝶谷
也已經到過蝴蝶谷與胡青牛、王難姑對話
在劇情上已經觸發知道了
金花婆婆便是紫衫龍王的劇情
可是即便如此，再去跟金花婆婆對話
無也法觸發紫衫龍王回歸明教的劇情
只能一直跟金花婆婆決鬥
請問這個部份要怎麼解決？

**Reply by MephistoSatanDevil** [reply]
https://github.com/jynew/jynew/pull/900
这次更新已经修复

**Comment by supermansaga** [comment]
谢谢。同是天涯沦落人。别人的3/10打包版可以覆盖Steam文件。终于可以正常通关了.

发自我的iPhone

**Comment by supermansaga** [comment]
有可能同样的操作去重制1997年的智冠游戏《神雕侠侣》吗？谢谢。那个的音乐一样经典。
